examples/default/version.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_version = ">= 1.0.0"
+  experiments      = [module_variable_optional_attrs]
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.41.1"
+    }
+    tls = {
+      source = "hashicorp/tls"
+      version = "3.4.0"
```

Comment:
Leave a comment above this saying:
`# The tls provider is not actually required by the module itself, just this example, so OK to use ">=" here instead of locking into a version`

and then update the version to `>= 3.4.0`